package.json

```diff
@@ -59,7 +59,7 @@
     "@wordpress/hooks": "2.12.3",
     "@wordpress/jest-puppeteer-axe": "3.0.3",
     "@wordpress/plugins": "3.0.0",
-    "@wordpress/scripts": "15.0.1",
+    "@wordpress/scripts": "16.0.0",
```

Comment:
Updated `@wordpress/scripts` to v16 as it uses `puppeteer-core` v9, which has support for Apple M1.